### PR TITLE
[SDPAPC-3] Added logout button to site header.

### PR DESCRIPTION
**Pull request by theleebriggs**
**JIRA issue:** https://digital-engagement.atlassian.net/projects/SDPAPC/issues/SDPAPC-3

### Changed

1. Added boolean `showLogout` prop to `RplSiteHeader`
2. This conditionally renders a logout button in the header
3. When the button is clicked `RplSiteHeader` emits a `logout` event to be consumed

### Screenshots

screen shot 2019-01-09 at 2 36 11 pm
screen shot 2019-01-09 at 2 36 31 pm

### CODE REVIEW

File: packages/Organisms/SiteHeader/index.vue [hunk 1]
```diff
@@ -68,6 +78,7 @@ import RplMenu from './menu'
 import RplSearch from './search'
 import RplIcon from '@dpc-sdp/ripple-icon'
 import RplLink from '@dpc-sdp/ripple-link'
+import RplButton from '@dpc-sdp/ripple-button'
```
Comment (tim-yao) [inline packages/Organisms/SiteHeader/index.vue]: Hey @theleebriggs You should able to remove this `RplButton` from here as it didn't get used as I can see.

File: packages/Organisms/SiteHeader/index.vue [hunk 2]
```diff
@@ -78,13 +89,15 @@ export default {
...
+    showSearch: { default: false, type: Boolean },
+    showLogout: { default: false, type: Boolean }
   },
   components: {
     RplIcon,
     RplLink,
     RplMenu,
-    RplSearch
+    RplSearch,
+    RplButton
```
Comment (tim-yao) [inline packages/Organisms/SiteHeader/index.vue]: Remove this line as well.

File: packages/Organisms/SiteHeader/index.vue (hunk 1 above)
Comment (theleebriggs) [reply packages/Organisms/SiteHeader/index.vue]: Fixed

File: packages/Organisms/SiteHeader/index.vue (hunk 2 above)
Comment (theleebriggs) [reply packages/Organisms/SiteHeader/index.vue]: Fixed

**Pull request merged**